.ci/pipeline/ci.yml

```diff
@@ -57,6 +57,65 @@ jobs:
       pip install isort black[jupyter]==24.1.1
       isort --profile black --check . && black --check .
     displayName: 'Linting'
+- job: LinuxCondaRecipe
+  dependsOn: Lint
+  timeoutInMinutes: 120
+  strategy:
+    matrix:
+      Python3.12_Numpy1.26:
+        PYTHON_VERSION: '3.12'
+        NUMPY_VERSION: '1.26'
+      Python3.9_Numpy1.22:
+        PYTHON_VERSION: '3.9'
+        NUMPY_VERSION: '1.22'
```

Comment:
Any justification for pinning such numpy versions?

Reply:
https://github.com/intel/scikit-learn-intelex/pull/1992#discussion_r1804754817